### PR TITLE
internal: add pthread compatibility header

### PR DESCRIPTION
**Pull request by mbouron-gpsw**

### CODE REVIEW

File: .github/workflows/ci_msvc.yml [hunk 1]
```diff
@@ -47,7 +47,6 @@ jobs:
           pip.exe install meson
           cd /mnt/c/vcpkg
           ./vcpkg.exe install --triplet ${{env.VCPKG_TRIPLET}} ${{env.VCPKG_INSTALL_PACKAGES}}
-          ./vcpkg.exe integrate install
```
Comment (ubitux) [inline .github/workflows/ci_msvc.yml:50]: can we have an explanation on why this goes away?

Comment (mbouron-gpsw) [reply .github/workflows/ci_msvc.yml:50]: Done.

**Review by cboesch-gpsw — APPROVED** [review]

**Review by jmoguillansky-gpsw — APPROVED** [review]

File: libnodegl/android_handlerthread.c [hunk 2]
```diff
@@ -19,13 +19,13 @@
...
  */
 
-#include <pthread.h>
 #include <stdlib.h>
 
 #include "android_looper.h"
 #include "android_handler.h"
 #include "android_handlerthread.h"
 #include "memory.h"
+#include "pthread_compat.h"
```
Comment (jmoguillansky-gpsw) [inline libnodegl/android_handlerthread.c:28]: we can still use pthread.h for android-specific code?

Comment (mbouron-gpsw) [reply libnodegl/android_handlerthread.c:28]: IMHO, it eases maintenance as you only have to look for pthread_compat.h across the code base and you know that <pthread.h> is only included in pthread_compat.h.

**Pull request merged**